src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/MultiPropertyDescriptorGridEntry.cs

```
@@ -153,20 +156,15 @@ protected override bool CreateChildren(bool diffOldChildren)
         }
```

Consider adding Debug.Fail on line 154 where you catch all exceptions.